script/Ciel/Day 7/07_01_CIEL07_6.txt

```diff
@@ -1031,7 +1032,7 @@ It doesn't seem like there are any other vampires here.
 [sha:c2bb4ee91c4371e4b9ee690c097ab7f7c5082653]{
 -- Page 569, Offset 16175.
 -- ロアとやらが健在ならマンションの住人すべてが襲いかかってきたかもしれないが、今はこれだけだ。
-It's possible that if this Roa person had still been alive and well the entire apartment complex would have swarmed us,
+It's possible that if this Roa person had still been alive and well, the entire apartment complex would have swarmed us,
```

Comment:
This one breaks the flow weirdly imo

Reply:
I thought it was too wordy and needed a break somewhere